src/olympia/migrations/1053-remove-compatibility-related-tables.sql

```diff
@@ -0,0 +1,2 @@
+DROP TABLE 'compatibility_reports';
+DROP TABLE 'compat_totals';
```

Comment:
New line at the end of the file please :)

Reply:
Done!